x-pack/legacy/plugins/actions/server/shim.ts

```diff
@@ -101,6 +106,9 @@ export function shim(
   const initializerContext: ActionsPluginInitializerContext = {
     logger: newPlatform.coreContext.logger,
     config: {
+      kibana$: Rx.of({
+        index: server.config().get('kibana.index'),
+      }),
```

Comment:
@rudolf is this the proper assumption based on https://github.com/elastic/kibana/issues/46240#issuecomment-540954412? I recall reading and mention it will eventually move to saved objects.

Reply:
Yes that's our latest design. We initially thought `kibana.index` belongs to the Saved Objects domain and so should be exposed from there, but the way actions and spaces (and maybe others) use `kibana.index` it's become more like a tenant ID for having multiple kibana clusters share the same ES cluster. So it feels more like a global config than something limited to saved objects.

Reply:
Ya, I'll be needing it for the event log, which is independent of actions and alerts (but those USE event log), and it generates ES resources that should match `config.get('kibana.index')`.  So, having it in SO would be a little weird, since the event log doesn't use SO, and would only pre-req it to get the name of the kibana index.